modules/kryo/pom.xml

```diff
@@ -22,7 +22,7 @@
     <relativePath>../../pom.xml</relativePath>
   </parent>
   <artifactId>fluo-recipes-kryo</artifactId>
-  <name>Fluo Recipes Kryo</name>
+  <name>Apache Fluo (incubating) Recipes for Kryo</name>
```

Comment:
Are these "for" Kryo, or "using" Kryo? I wasn't 100% confident with my suggestion to make this particular change.